ganttproject/src/net/sourceforge/ganttproject/GPTreeTableBase.java

```diff
@@ -528,10 +536,80 @@ protected void initTreeTable() {
     doInit();
   }
 
+  private static Comparator<Task> beginComparatorAsc = new Comparator<Task>() {
```

Comment:
Good point. You can use the same approach which is used with isEditable. Look at `GanttTreeTableModel` constructor.